auto_scan.py

```diff
@@ -564,6 +616,11 @@ def main(args):
         action='store_true',
         help='Only list active containers for integrated/specified registries (no scans)'
     )
+    parser.add_argument(
```

Comment:
I really like having this be a separate, explicit argument. I was thinking this level of explicit would be a must as soon as I read the body of the PR.